tests/lib/cli.js

```diff
@@ -570,21 +570,21 @@ describe("cli", () => {
 
                 it(`should only print error with configType:${configType}`, async () => {
                     const filePath = getFixturePath("single-quoted.js");
-                    const cliArgs = `--no-ignore --quiet  -f compact --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
+                    const cliArgs = `--no-ignore --quiet  -f stylish --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
```

Comment:
This test aims to assert that warnings won't be printed when the `--quiet` option is used. However, the test doesn't work as intended because there are no `no-unused-vars` problems in the `single-quoted.js` fixture file, so the test would pass even without the `--quiet` option (or if the `--quiet` option doesn't work as intended, which is the point of the test) because there are no warnings at all.

To fix this, we can use the `no-undef` rule, which actually does report a problem in the fixture file.

```suggestion
                    const cliArgs = `--no-ignore --quiet -f stylish --rule 'quotes: [2, double]' --rule 'no-undef: 1' ${filePath}`;
```

...